tests/queries/0_stateless/01602_insert_into_table_function_cluster.sql

```diff
@@ -1,22 +1,23 @@
 DROP TABLE IF EXISTS default.x;
+DROP TABLE IF EXISTS default.y;
 
 CREATE TABLE default.x AS system.numbers ENGINE = MergeTree ORDER BY number;
```

Comment:
There is still explicit database in the test, and this will not work, since each test is run with unique database name